packages/search-ui/src/__tests__/SearchDriver.test.js

```diff
@@ -437,3 +441,69 @@ describe("_updateSearchResults", () => {
     });
   });
 });
+
+describe("When multiple actions are called", () => {
+  it("Will batch them into a single API call", () => {
+    const { driver, mockApiConnector } = setupDriver();
+    driver.setSearchTerm("term");
+    driver.addFilter("field1", "value1");
+    driver.addFilter("field2", "value2");
+    driver.addFilter("field3", "value3");
+    jest.runAllTimers();
+    expect(getSearchCalls(mockApiConnector)).toHaveLength(1);
+    expect(getSearchCalls(mockApiConnector)[0][0].filters).toEqual([
+      {
+        field: "field1",
+        values: ["value1"],
+        type: "all"
+      },
+      {
+        field: "field2",
+        values: ["value2"],
+        type: "all"
+      },
+      {
+        field: "field3",
+        values: ["value3"],
+        type: "all"
+      }
+    ]);
+  });
+
+  describe("and setSearchTerm is called with a debounce", () => {
```

Comment:
Nit: is the "and" at the front of this context necessary? I think you can remove it for just "setSearchTerm is called with a debounce" and it still makes sense.